packages/@core.keystone/schema.js

```diff
@@ -182,6 +182,14 @@ async function getByCondition (schemaName, condition) {
     else return null
 }
 
+/**
+ * This function returns raw data from a database table. This may be necessary to optimize queries or to write custom mutations to reuse existing Apollo resolvers because they work only with the raw database data.
```

Comment:
It's clear, that we are commenting this function ;)

```suggestion
 * Returns raw data from a database table. This may be necessary to optimize queries or to write custom mutations to reuse existing Apollo resolvers because they work only with the raw database data.
```

Reply:
fixed